game/src/ecs/components/PositionComponent.java

```diff
@@ -19,6 +19,14 @@ public PositionComponent(Entity entity, Point point) {
         this.position = point;
     }
 
+    /**
+     * @param entity associated entity
+     */
+    public PositionComponent(Entity entity) {
+        super(entity);
+        this.position = null;
```

Comment:
Die total intuitive Syntax dafür ist: 

```suggestion
        this.position =  ECS.currentLevel.getRandomTile(LevelElement.FLOOR).getCoordinate().toPoint();
```